package.json

```diff
@@ -29,12 +29,12 @@
   },
   "homepage": "https://github.com/HMPO/hmpo-reqres",
   "dependencies": {
-    "express": "^4.19.2",
-    "sinon": "^18.0.0"
+    "express": "^4.21.0",
+    "sinon": "^19.0.2"
   },
   "devDependencies": {
```

Comment:
Need `husky` added and set up with pre-push hook

Reply:
We're still missing the husky `prepare` package script meaning `npx husky init` hasn't been run